catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -78,6 +79,9 @@ define([
         handleRun: function(){
             this.model.startSearch();
         },
+        handleHitCount: function(){
+           this.model.getHitCount();
```

Comment:
✏️  I would prefer not to have a `getHitCount` method and just call `this.model.startSearch({hitCountOnly: true})` here.

`getHitCount` doesn't align with the verbiage used in `startSearch`, so it feels like it's doing an unrelated operation. At first glance sounds like it's just a getter accessing some data that's already been fetched, not performing a new search.

Reply:
Yeah that's fine. I did this early on before I knew it was going to be that simple to create a hit count only query.